src/member/apps.py

```diff
@@ -3,3 +3,9 @@
 
 class MemberConfig(AppConfig):
     name = 'member'
+
+    def ready(self):
+        # import signal handlers
+        # (every models imported inside handlers will be instantiated as
+        # soon as the registry is fully populated.)
+        import member.signals.handlers  # noqa
```

Comment:
This import looks a little bit magic as what we really want here is to make the connection of the signal and the handler. The `@receiver` decorator is not very clear on its side effect.

I suggest writing `post_delete.connect(ensure_pair_remove, sender=ClientScheduledStatus)` in the ready method to improve the readability.